rules/opentitan/splice.bzl

```diff
@@ -106,6 +106,11 @@ def _bitstream_splice_impl(ctx):
     exec_env = ctx.attr.exec_env[ExecEnvInfo]
     src = ctx.file.src if ctx.file.src else exec_env.base_bitstream
 
+    # If configured to skip bitstream loading, do not splice and forward the file:
+    # opentitantool skips loading if the file is named `skip.bit`.
+    if src.path.endswith("skip.bit"):
```

Comment:
Hm... Special file names feel a little awkward compared to using configuration from a build setting and provider.

Reply:
Yes I agree, I am still planning on cleaning that up. In the mean time, I am keeping the PR alive just in case some people need it (since skip is broken).

It would also be nice if we had a simple way of passing a specific bitstream to bazel so you can e.g. get a bitstream build from the CI and let bazel splice and use it.

Reply:
@a-will what do you think of this new implementation?

Reply:
> It would also be nice if we had a simple way of passing a specific bitstream to bazel so you can e.g. get a bitstream build from the CI and let bazel splice and use it.

I'm not sure you're going to get "simple" with that particular function, hehe. However, it's possible to do now by creating a test bitstream cache, placing the desired bitstream output (the tar file with the manifest) in a subdirectory with some name `<test-bit-dir>`, pointing `BAZEL_BITSTREAMS_CACHE` to the bitstream cache directory, and modifying the `BITSTREAM` environment variable to contain something like "--offline `<test-bit-dir>`"

This is exactly what CI does, and it's how I test against bitstreams that haven't entered the cache (either built by me or CI).

Reply:
I know it is doable but this is not exactly "simple". I was thinking doing something like:
```bazel test --define bitstream_tgz=/path/to/bitstream.tar ...`
and it automatically takes care of extracting the bitstream and mmi files.

Reply:
yes agreed